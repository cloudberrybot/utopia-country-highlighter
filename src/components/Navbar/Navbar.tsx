import { Logo } from '../Logo';
import React from 'react';

function Navbar() {
  return (
    <div className="grid grid-cols-3 md:grid-cols-4 gap-4 h-[100px] md:h-[183px] bg-navbar items-center">
      <div className="col-span-2 md:col-start-2 md:col-end-4 md:justify-self-center">
        <div className="text-lg md:text-4xl lg:text-5xl text-white font-sans pl-6 min-w-full whitespace-nowrap">Utopia Country Highlighter</div>
      </div>
      <div className="relative w-14 h-14 md:w-24 md:h-24 mr-2 md:mr-6 justify-self-end">
        <Logo />
      </div>
    </div>
  )
}

export default Navbar;