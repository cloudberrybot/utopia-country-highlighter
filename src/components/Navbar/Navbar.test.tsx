import '@testing-library/jest-dom'

import Navbar from './Navbar';
import { render } from '@testing-library/react';

test('renders europe react link', () => {
  const { asFragment } = render(
    <Navbar />
  );

  expect(asFragment()).toMatchSnapshot();
});
