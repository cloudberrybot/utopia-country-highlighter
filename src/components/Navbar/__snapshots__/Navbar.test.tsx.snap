// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders europe react link 1`] = `
<DocumentFragment>
  <div
    class="grid grid-cols-3 md:grid-cols-4 gap-4 h-[100px] md:h-[183px] bg-navbar items-center"
  >
    <div
      class="col-span-2 md:col-start-2 md:col-end-4 md:justify-self-center"
    >
      <div
        class="text-lg md:text-4xl lg:text-5xl text-white font-sans pl-6 min-w-full whitespace-nowrap"
      >
        Utopia Country Highlighter
      </div>
    </div>
    <div
      class="relative w-14 h-14 md:w-24 md:h-24 mr-2 md:mr-6 justify-self-end"
    >
      <img
        alt="logo"
        class="rounded-full"
        src="logo.png"
      />
      <span
        class="flex absolute h-5 w-5 top-0 md:top-2 right-1 md:right-2 -mt-1 -mr-1"
      >
        <span
          class="relative inline-flex rounded-full h-4 w-4 bg-green-500"
        />
      </span>
    </div>
  </div>
</DocumentFragment>
`;
